### PR TITLE
remove all *_currency fields

### LINKED ISSUE #128: Remove all fields with *_currency.
All fields with *_currency in them are a bug. They should not be part of the API.

### PR DESCRIPTION
**Pull request by jtprince**
All fields with `*_currency` in them are a bug.  They should not be part of the API.  I removed all references to them, but someone may want to check my work (I didn't run any code to generate output).

I just put in a fix so that these fields should not show up again in API output (see https://github.com/CruxConnect/crux-api/pull/1063).

Fixes #128 

LOE: 1

### CODE REVIEW

**Comment by edje-jeter** [comment]
Why are currency fields a bug? How will the UI know what currency to display if there isn't a currency field?

**Comment by edje-jeter** [comment]
Follow up, @jtprince: Did we decide somewhere to only use USD and I just missed the decision? If we are always and only doing USD then it makes sense to remove currency fields.

**Comment by Gidgidonihah** [comment]
@nickcoleman This PR needs to have the updated master paw file merged into it and have the changes applied to its docs.

**Comment by nickcoleman** [comment]
- [ ] _product_retailer/get_catalog_detail.md
- [ ] _product_retailer/get_catalog_detail.md
- [ ] _product_retailer/get_item_detail.md
- [ ] _product_retailer/get_item_export.md
- [ ] _product_retailer/get_items.md
- [ ] _product_retailer/get_items.md
- [ ] _product_retailer/get_skus.md
- [ ] _product_retailer/search_items.md
- [ ] _product_supplier/catalog_add_item.md
- [ ]  _product_supplier/create_catalog.md
- [ ] _product_supplier/create_item.md
- [ ] _product_supplier/create_sku.md

File: _product_retailer/get_catalog_detail.md [hunk 1]
```diff
@@ -1,44 +1,47 @@
 ---
-title: /products/catalogs/&ltcatalog_uuid&gt/
+title: /products/catalogs/38504ca3-27ea-4478-85a2-25f01cde1652/
```
Comment (Gidgidonihah) [inline _product_retailer/get_catalog_detail.md]: We won't want to change this.

File: _product_retailer/get_catalog_detail.md [hunk 2]
```diff
@@ -1,44 +1,47 @@
...
 name: Get Catalog Detail
-position: 2.01
-visibility: public
+position: 1.01
+visibility: Internal
```
Comment (Gidgidonihah) [inline _product_retailer/get_catalog_detail.md]: This doesn't seem right.

File: _product_retailer/get_catalogs.md [hunk 3]
```diff
@@ -1,90 +1,57 @@
 ---
 title: /products/catalogs/
 name: Get Catalogs
-position: 2.00
-visibility: public
+position: 1.0
+visibility: Internal
```
Comment (Gidgidonihah) [inline _product_retailer/get_catalogs.md]: Seems wrong.

File: _product_retailer/get_item_detail.md [hunk 4]
```diff
@@ -1,252 +1,160 @@
 ---
-title: /products/items/&ltitem_uuid&gt/
+title: /products/items/
 name: Get Item Detail
-position: 2.20
-visibility: public
+position: 1.04
+visibility: Internal
```
Comment (Gidgidonihah) [inline _product_retailer/get_item_detail.md]: Hmm

File: _product_retailer/get_item_detail.md [hunk 5]
```diff
@@ -1,252 +1,160 @@
 ---
-title: /products/items/&ltitem_uuid&gt/
+title: /products/items/
```
Comment (Gidgidonihah) [inline _product_retailer/get_item_detail.md]: We lost the param. Also it should be `&lt;item_uuid&gt;` (note the added `;`s)

File: _product_retailer/get_catalog_detail.md (hunk 1 above)
Comment (Gidgidonihah) [reply _product_retailer/get_catalog_detail.md]: Also, add missing `;`s: `&lt;catalog_uuid&gt;`

Comment (nickcoleman) [reply _product_retailer/get_catalog_detail.md]: crap ... thanks

File: _product_retailer/get_catalog_detail.md [hunk 6]
```diff
@@ -1,45 +1,47 @@
...
-    "default_shipping_cost_currency": "USD"
+    "created": "2018-04-06T01:12:09.804995Z",
+    "last_updated": "2018-04-06T01:12:18.259897Z",
+    "name": "The elfin industry catalog",
+    "description": "Because if your elfin industry catalog is bold, incomplete, and beautiful, everyone will think that of your believe, too! It's clear, crisp, and guaranteed! Oh, no you don't!  Our elfin industry catalog kicks the earthy competition in the smile! Because we care about how your elfin industry catalog looks! You know what's accurate about elfin industry catalog? Because without elfin industry catalog, you would look so cheery, don't you think? Be the kind of person your mother wanted you to me. All your wildest dreams would come true. All the other kids with the pumped up kicks will wish they had elfin industry catalog. I like, it, I love it, I want some more of it. When it's all said and done, there's still elfin industry catalog. Still. You know you want it."
```
Comment (jtprince) [inline _product_retailer/get_catalog_detail.md]: If we are using populatedb auto-generated descriptions, we may want to simplify and business-ify them.  I never intended those auto-generated descriptions to be public-facing. @nickcoleman

File: _product_retailer/get_catalog_detail.md (hunk 2 above)
Comment (nickcoleman) [reply _product_retailer/get_catalog_detail.md]: agree ... just figured out why ... I'd set up an internal test account when messing with discovery 😢 and was using it.  I'll pull it from the paw environment.

File: _product_retailer/get_catalog_detail.md (hunk 6 above)
Comment (nickcoleman) [reply _product_retailer/get_catalog_detail.md]: I'll shorten it

File: _product_retailer/get_catalogs.md (hunk 3 above)
Comment (nickcoleman) [reply _product_retailer/get_catalogs.md]: agree .. see above

File: _product_retailer/get_item_detail.md (hunk 4 above)
Comment (nickcoleman) [reply _product_retailer/get_item_detail.md]: agree ... see above

File: _product_retailer/get_item_detail.md (hunk 5 above)
Comment (nickcoleman) [reply _product_retailer/get_item_detail.md]: 👍

File: _product_retailer/get_catalog_detail.md [hunk 7]
```diff
@@ -1,45 +1,47 @@
 ---
-title: /products/catalogs/&ltcatalog_uuid&gt/
+title: /products/catalogs/&lt;uuid&gt;/
```
Comment (Gidgidonihah) [inline _product_retailer/get_catalog_detail.md]: Was `catalog_uuid` is now `uuid` and doesn't match the param as specified below.

File: _product_retailer/get_catalog_detail.md [hunk 8]
```diff
@@ -1,45 +1,47 @@
...
 name: Get Catalog Detail
-position: 2.01
-visibility: public
+position: 1.01
+visibility: Public
```
Comment (Gidgidonihah) [inline _product_retailer/get_catalog_detail.md]: Have you checked that case doesn't matter?

File: _product_retailer/get_catalog_detail.md (hunk 7 above)
Comment (nickcoleman) [reply _product_retailer/get_catalog_detail.md]: fixed

File: _product_retailer/get_catalog_detail.md (hunk 8 above)
Comment (nickcoleman) [reply _product_retailer/get_catalog_detail.md]: it doesn't appear to matter.  but, lower cased it for consistency

**Pull request merged**